libraries/typescript/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import * as ts from "@testable-ts/v3.0";
```

Comment:
This has to be a 3.0 version of typescript because newer typescript versions make use of non-downward-compatible typescript features in their types.

So testing 3.0 with types of 3.7 does not work, while testing 3.7 with types from 3.0 works just fine - especially as these types are just being used to interpret the code of `tsd` itself (I am not sure but I believe outside of the test suite this would not play any role at all).